wagtail/contrib/search_promotions/models.py

```diff
@@ -91,26 +146,70 @@ class Meta:
 
 
 class SearchPromotion(models.Model):
+    """
+    Model for search promotion.
+
+    The promotion is for either a page or external url, not both.
+
+    Attributes
+       - query (models.ForeignKey): Matching query object.
+       - page (wagtail.models.Page): Matching page, if any.
+       - external_link (models.URLField): Matching external url, if any.
+       - external_title (models.CharField) Title text, if external url.
+       - description (models.TextField): Text description of search promotion.
+       - sort_order (int): Ordinal of search promotion.
+    """
+
     query = models.ForeignKey(
         Query, db_index=True, related_name="editors_picks", on_delete=models.CASCADE
     )
     page = models.ForeignKey(
-        "wagtailcore.Page", verbose_name=_("page"), on_delete=models.CASCADE
+        "wagtailcore.Page",
+        verbose_name=_("page"),
+        on_delete=models.CASCADE,
+        null=True,
+        blank=True,
     )
-    sort_order = models.IntegerField(null=True, blank=True, editable=False)
+    external_link = models.URLField(_("Url"), blank=True)
```

Comment:
```suggestion
    external_url = models.URLField(_("URL"), blank=True)
```

OR

```suggestion
    external_link_url = models.URLField(_("URL"), blank=True)
```

As noted above, user facing text should use `URL` (it's an abbreviation), and the thing as a whole should be `link`.